server/svix-server/src/v1/endpoints/endpoint/mod.rs

```diff
@@ -348,14 +355,32 @@ impl From<endpoint::Model> for EndpointOut {
             disabled: model.disabled,
             event_types_ids: model.event_types_ids,
             channels: model.channels,
-
-            id: model.id,
             created_at: model.created_at.into(),
             updated_at: model.updated_at.into(),
         }
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, ModelOut)]
+#[serde(rename_all = "camelCase")]
+pub struct EndpointOut {
+    #[serde(flatten)]
+    pub ep: EndpointOutNoMetadata,
+    pub id: EndpointId,
+    pub metadata: Metadata,
```

Comment:
This shouldn't be returned normally, only when metadata is specifically asked for.

Reply:
In our standard API we always return this field. We should definitely be consistent (I also don't see the value in hiding this field normally regardless)

Reply:
Oh we do, I thought we didn't.